estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': "Real Estate",
+    'version': '1.0',
+    'depends': ['base'],
+    'author': "drat-odoo",
+    'category': 'Category',
+    'description': """
+    This is base version of estate
+    """,
```

Comment:
```suggestion
    'description': """
This is base version of estate
    """,
```
Description generally describe the brief of what this module provides, update as per your understanding.